frontend/src/app/globals.css

```
@@ -79,7 +79,13 @@ body {
   background-attachment: fixed;
   overflow-x: hidden;
   font-family:
-    'PingFang SC',
+    'Google Sans Flex',
+    'Google Sans',
+    'Noto Sans Simplified Chinese',
+    'Noto Sans SC',
+    'Inter',
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)